### DISCUSSION TITLE
Gmail mobile webmail not supports `Display: none`

### DISCUSSION THREAD

**Discussion by DSchuurmanBadge** [bootstrap-email/bootstrap-email#171] [Help]
Does anyone no the alternative of `display: none` with bootstrap e-mail? I need to show an image only on desktop, but show nothing on mobile. The only e-mail client where this does not work is in Gmail mobile webmail.
The classes are: 
- `d-none`
- `d-lg-inline`

**Comment by stuyam** [comment]
If anyone has any work arounds for gmail mobile webview let me know. From everything I have been able to see there has been no way for it to work in the gmail mobile webview as they support very little media query or other features to get around this.
